tests/network/salt_minion.pm

```diff
@@ -0,0 +1,70 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016-2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test Salt stack on two machines. This machine is running
+#  salt-minion only and here we test the end result of master operations.
+# Maintainer: Pavel Dostal <pdostal@suse.cz>
+
+use base "consoletest";
+use strict;
+use testapi;
+use lockapi;
+use mm_network;
+use utils qw(zypper_call pkcon_quit systemctl);
+use version_utils 'is_sle';
+use registration 'add_suseconnect_product';
+
+sub run {
+    my ($self) = @_;
+    my $hostname = get_var('HOSTNAME');
+    select_console 'root-console';
+
+    # Configure static network, disable firewall
+    configure_default_gateway;
+    systemctl 'stop ' . $self->firewall;
+    configure_static_ip('10.0.2.102/24');
+    configure_static_dns(get_host_resolv_conf());
+
+    # Set the hostname to identify both minions
+    assert_script_run "hostnamectl set-hostname $hostname";
+    assert_script_run "hostnamectl status|grep $hostname";
+    assert_script_run "hostname|grep $hostname";
+
+    # Install the salt minion, set the address of the master and start it
+    zypper_call('in salt-minion');
+    assert_script_run('sed -i -e "s/#master: salt/master: 10.0.2.101/" /etc/salt/minion');
+    assert_script_run("cat /etc/salt/minion | grep 'master: '");
```

Comment:
this would have purpose when it would check expected value e.g. `grep ^master: 10.0.2.101`